src/test/scala/ai/privado/languageEngine/go/passes/orm/GormParserTest.scala

```diff
@@ -0,0 +1,107 @@
+/*
+ * This file is part of Privado OSS.
+ *
+ * Privado is an open source static code analysis tool to discover data flows in the code.
+ * Copyright (C) 2022 Privado, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * For more information, contact support@privado.ai
+ *
+ */
+package ai.privado.languageEngine.go.passes.orm
+
+import ai.privado.languageEngine.go.tagger.GoTaggingTestBase
+import ai.privado.model.*
+import ai.privado.semantic.Language.*
+import io.joern.x2cpg.X2Cpg
+import io.shiftleft.semanticcpg.language.*
+import io.shiftleft.semanticcpg.layers.LayerCreatorContext
+import io.joern.dataflowengineoss.layers.dataflows.{OssDataFlow, OssDataFlowOptions}
+
+class GormParserTest extends GoTaggingTestBase {
```

Comment:
GoTaggingTestBase and be modified to have `withOSSDataflow` which takes boolean, and pass can be applied accordingly

Reply:
Found dataflow isn't necessary for these test case. Removed it under: ea366f1f6499723e5f9c5835a13da604a66f9a17